src/index.ts

```diff
@@ -11,6 +11,38 @@ export class GraphQLClient {
     this.options = options || {}
   }
 
+  async rawRequest<T extends any>(
+    query: string,
+    variables?: Variables,
+  ): Promise<T> {
```

Comment:
Please adjust to `Promise<{ data?: T, extensions?: any, errors?: GraphQLError }>`